--- origsrc/rkward-0.4.9a/admin/acinclude.m4.in	2007-05-06 17:28:38.000000000 -0500
+++ src/rkward-0.4.9a/admin/acinclude.m4.in	2008-04-22 03:37:18.328125000 -0500
@@ -1949,8 +1949,8 @@
 AC_DEFUN([CHECK_LIBRSO],
 [
 AC_MSG_CHECKING(for libR.so)
-if test -f "$R_HOMEDIR/lib/libR.so"; then
-   AC_MSG_RESULT($R_HOMEDIR/lib/libR.so)
+if test -f "$R_HOMEDIR/lib/libR.dll.a"; then
+   AC_MSG_RESULT($R_HOMEDIR/lib/libR.dll.a)
 else
    AC_MSG_ERROR([
    $R_HOMEDIR/lib/libR.so does not exist.
@@ -1965,9 +1965,9 @@
 AC_DEFUN([CHECK_RLAPACK],
 [
 AC_MSG_CHECKING(whether we should link against libRlapack.so)
-if test -f "$R_HOMEDIR/lib/libRlapack.so"; then
-   AC_MSG_RESULT(yes $R_HOMEDIR/lib/libRlapack.so exists)
-   R_LAPACK_FLAG="-lRlapack -lgfortran"
+if test -f "$R_HOMEDIR/lib/libRlapack.dll.a"; then
+   AC_MSG_RESULT(yes $R_HOMEDIR/lib/libRlapack.dll.a exists)
+   R_LAPACK_FLAG="-lRlapack"
 else
    AC_MSG_RESULT(no $R_HOMEDIR/lib/libRlapack.so does not exist)
    R_LAPACK_FLAG=""
@@ -1981,8 +1981,8 @@
 AC_DEFUN([CHECK_RBLAS],
 [
 AC_MSG_CHECKING(whether we should link against libRblas.so)
-if test -f "$R_HOMEDIR/lib/libRblas.so"; then
-   AC_MSG_RESULT(yes $R_HOMEDIR/lib/libRblas.so exists)
+if test -f "$R_HOMEDIR/lib/libRblas.dll.a"; then
+   AC_MSG_RESULT(yes $R_HOMEDIR/lib/libRblas.dll.a exists)
    R_BLAS_FLAG="-lRblas"
 else
    AC_MSG_RESULT(no $R_HOMEDIR/lib/libRblas.so does not exist)
--- origsrc/rkward-0.4.9a/rkward/Makefile.am	2007-11-20 09:27:36.000000000 -0600
+++ src/rkward-0.4.9a/rkward/Makefile.am	2008-04-22 03:37:18.343750000 -0500
@@ -67,7 +67,7 @@
 
 # all the perl script does is fill in the placeholder for R_HOMEDIR and friends
 rkward: makerkwardscript.pl rkward.sh.template
-	perl ./makerkwardscript.pl rkward.sh.template "$(R_HOMEDIR)" > rkward; \
+	perl $(srcdir)/makerkwardscript.pl $(srcdir)/rkward.sh.template "$(R_HOMEDIR)" > rkward; \
 	chmod 744 rkward
 
 CLEANFILES = rkward
--- origsrc/rkward-0.4.9a/rkward/rbackend/rpackages/Makefile.am	2007-06-01 05:09:58.000000000 -0500
+++ src/rkward-0.4.9a/rkward/rbackend/rpackages/Makefile.am	2008-04-22 03:37:18.343750000 -0500
@@ -1,6 +1,6 @@
-SUBDIRS = rkward
+DIST_SUBDIRS = rkward
 
-all:
+all: pkg-stamp
 	-${R_HOMEDIR}/bin/R CMD build rkward
 install:
 	-mkdir -p tmp ; \
@@ -17,3 +17,8 @@
 distclean:
 	-rm -f rkward_*.tar.gz
 	-rm -rf tmp
+pkg-stamp:
+	if ! test -e ./rkward/DESCRIPTION; then \
+		(mkdir -p rkward && cd rkward && lndir $(srcdir)/rkward . ) ; \
+	fi && \
+	touch $@
--- origsrc/rkward-0.4.9a/rkward/rkward.kidl	2008-04-20 13:55:05.000000000 -0500
+++ src/rkward-0.4.9a/rkward/rkward.kidl	1969-12-31 18:00:00.000000000 -0600
@@ -1,20 +0,0 @@
-<!DOCTYPE DCOP-IDL><DCOP-IDL>
-<SOURCE>./rkward.h</SOURCE>
-<INCLUDE>config.h</INCLUDE>
-<INCLUDE>dcopobject.h</INCLUDE>
-<CLASS>
-    <NAME>RKWardDCOPInterface</NAME>
-    <SUPER>DCOPObject</SUPER>
-    <FUNC>
-        <TYPE>void</TYPE>
-        <NAME>openHTMLHelp</NAME>
-        <ARG><TYPE  qleft="const" qright="&amp;">QString</TYPE><NAME>url</NAME></ARG>
-     </FUNC>
-</CLASS>
-<INCLUDE>kapp.h</INCLUDE>
-<INCLUDE>kmdimainfrm.h</INCLUDE>
-<INCLUDE>kaccel.h</INCLUDE>
-<INCLUDE>kaction.h</INCLUDE>
-<INCLUDE>kurl.h</INCLUDE>
-<INCLUDE>kparts/partmanager.h</INCLUDE>
-</DCOP-IDL>
